ppr-ui/tests/unit/ManufacturerMakeModel.spec.ts

```diff
@@ -57,82 +32,110 @@ describe('ManufacturerMakeModel component - staff', () => {
     wrapper.find(getTestId('manufacturer-model')).exists()
   })
 
-  it('verifes Name of Manufactuer field is not disabled', async () => {
+  it('verifies Name of Manufacturer field is not disabled', async () => {
     expect(wrapper.find(getTestId('manufacturer-name')).attributes('disabled')).toBe(undefined)
   })
 
   it('show error messages for Name of Manufacturer field', async () => {
-    wrapper.find(getTestId('manufacturer-name')).setValue('x')
-    await nextTick()
-    wrapper.find(getTestId('manufacturer-name')).setValue('')
+    await wrapper.find('#manufacturer-name').setValue('x')
     await nextTick()
+    await wrapper.find('#manufacturer-name').setValue('')
     await nextTick()
+
     expect(wrapper.findAll(ERROR_MSG).length).toBe(1)
     expect(wrapper.findAll(ERROR_MSG).at(0).text()).toContain('Enter a manufacturer')
   })
 
   it('show error messages for Manufacturer Make Model inputs', async () => {
-    wrapper.find(getTestId('manufacturer-name')).setValue('x'.repeat(70))
-    wrapper.find(getTestId('manufacture-year')).setValue('x'.repeat(6))
-    wrapper.find(getTestId('manufacturer-make')).setValue('x'.repeat(70))
-    wrapper.find(getTestId('manufacturer-model')).setValue('x'.repeat(70))
+    await wrapper.find('#manufacturer-name').setValue('x'.repeat(70))
+    await wrapper.find('#manufacturer-year').setValue('x'.repeat(6))
+    await wrapper.find('#manufacturer-make').setValue('x'.repeat(70))
+    await wrapper.find('#manufacturer-model').setValue('x'.repeat(70))
     await nextTick()
     const messages = wrapper.findAll(ERROR_MSG)
     expect(messages.length).toBe(4)
   })
 
   it('show error messages for Year of Manufacture field', async () => {
-    const yearInputField = wrapper.find(getTestId('manufacture-year'))
+    // Set defaults to isolate year validation
+    const form = await wrapper.findComponent('.v-form')
+    await wrapper.find('#manufacturer-name').setValue('x')
+    await wrapper.find('#manufacturer-make').setValue('x')
+    await wrapper.find('#manufacturer-model').setValue('x')
```

Comment:
I wonder if we could get away with setting just one field... but as long as test works, we are ok..